<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerManagementWebView</class>
 <widget class="QWidget" name="Mer::Internal::MerManagementWebView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="Utils::StyledBar" name="styledBar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="homeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mer.qrc">
          <normaloff>:/mer/images/icon-s-sailfish-qtcreator.png</normaloff>:/mer/images/icon-s-sailfish-qtcreator.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Utils::FancyLineEdit" name="urlLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusBarLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>Utils::StyledBar</class>
   <extends>QWidget</extends>
   <header location="global">utils/styledbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mer.qrc"/>
 </resources>
 <connections/>
</ui>
